src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SimplifyBooleanExpressionCheck.java

```diff
@@ -48,18 +49,22 @@
  *     boolean a, b;
  *     Foo c, d, e;
  *
- *     if (!false) {}; // violation, can be simplified to true
+ *     if (!false) {};       // violation, can be simplified to true
  *
- *     if (a == true) {}; // violation, can be simplified to a
- *     if (a == b) {}; // OK
- *     if (a == false) {}; // violation, can be simplified to !a
+ *     if (a == true) {};    // violation, can be simplified to a
+ *     if (a == b) {};       // OK
+ *     if (a == false) {};   // violation, can be simplified to !a
  *     if (!(a != true)) {}; // violation, can be simplified to a
  *
  *     e = (a || b) ? c : d; // OK
- *     e = (a || false) ? c : d; // violation, can be simplified to a
+ *     e = (a || false) ? c : d;     // violation, can be simplified to a
  *     e = (a &amp;&amp; b) ? c : d; // OK
  *
- *  }
+ *     int s = 12;
+ *     boolean m = s > 1 ? true : false; // violation, can be simplified to s > 1
+ *     boolean f = a == b
+ *             ? false : s > 1; // OK
```

Comment:
it is better to change ok example to one from report `var == null ? false : var.someMethod()`

Reply:
done